bedrock/products/views.py

```diff
@@ -63,10 +63,7 @@ def vpn_landing_page(request):
     if entrypoint_experiment not in []:
         entrypoint_experiment = None
 
-    if ftl_file_is_active("products/vpn/landing-2023") and experience != "legacy":
-        template_name = "products/vpn/landing-refresh.html"
-    else:
-        template_name = "products/vpn/landing.html"
+    template_name = "products/vpn/landing-refresh.html"
```

Comment:
Since the `-refresh` suffix now seems redundant, is it worth renaming these files? Though maybe in a follow-up PR to avoid confusion and conflicts in this one.

Reply:
Yeah I wanted to try and keep the diff as clean and easily to review as I can. There’s quite a few things labelled refresh, so to be honest I’m tempted to leave those as is for the time being at least.